test/lock-parser/fixtures/restriction-overwrite/out.json

```diff
@@ -0,0 +1,232 @@
+{
+  "dependencies": [
+    {
+      "name": "FSharp.Core",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "4.0.0.1",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FAKE",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "4.62.1",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FSharp.Compiler.Service",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "2.0.0.6",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FSharp.Formatting",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "2.14.4",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": [
+        {
+          "name": "FSharp.Compiler.Service",
+          "options": {
+            "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+          },
+          "version": "2.0.0.6"
+        }
+      ]
+    },
+    {
+      "name": "FSharpVSPowerTools.Core",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "2.3",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": [
+        {
+          "name": "FSharpVSPowerTools.Core",
+          "options": {
+            "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+          },
+          "version": ">= 2.3 < 2.4"
+        },
+        {
+          "name": "FSharp.Compiler.Service",
+          "options": {
```

Comment:
why `options`, is it needed? Why not just `restriction`? Also would `target-framework-restriction` be more explicit as to what this field holds?

Reply:
In this case it is only a restriction. There are multiple options a dependency can have. e.g. `import_targets` or `framework` or `restriction`.